pkgs/development/python-modules/jsonschema/default.nix

```diff
@@ -1,31 +1,51 @@
-{ stdenv, buildPythonPackage, fetchPypi, python
-, nose, mock, vcversioner, functools32 }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, attrs
+, pyrsistent
+, setuptools
+, six
+, functools32
+, setuptools_scm
+, perf
+, twisted
+, python
+}:
 
 buildPythonPackage rec {
   pname = "jsonschema";
-  version = "2.6.0";
+  version = "3.0.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "00kf3zmpp9ya4sydffpifn0j0mzm342a2vzh82p6r0vh10cg7xbg";
+    sha256 = "0c0a81564f181de3212efa2d17de1910f8732fa1b71c42266d983cd74304e20d";
```

Comment:
I think @FRidh and the lot converged on python packages using the base32 hash

Reply:
Fixed.